codegen.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  schema: [
+    {
+      "http://localhost:9090/graphql/query": {
+        headers: {
+          cookie: process.env.REACT_APP_GQL_COOKIE,
+        },
+      },
+    },
+  ],
+  documents: ["./src/**/*.ts"],
+  overwrite: true,
+  generates: {
+    "./src/gql/generated/types.ts": {
```

Comment:
types are saved to `gql/generated/types.ts` if any one has a better name/location for this lmk.